automatic/paint.net/update.ps1

```diff
@@ -1,34 +1,41 @@
-﻿Import-Module AU
+Import-Module AU
 
-$releases = 'https://github.com/paintdotnet/release/releases'
+
+$domain   = 'https://github.com'
+$releases = "$domain/repos/paintdotnet/release/releases/latest"
+$owner = "paintdotnet"
+$repository = "release"
 
 function global:au_SearchReplace {
   @{
       ".\legal\VERIFICATION.txt" = @{
-        "(?i)(\s+x64:).*"            = "`${1} $($Latest.URL64)"
-        "(?i)(checksum64:).*"        = "`${1} $($Latest.Checksum64)"
-      }
+          "(?i)(\s+x32:).*"            = "`${1} $($Latest.URL32)"
+          "(?i)(checksum32:).*"        = "`${1} $($Latest.Checksum32)"
+          "(?i)(\s+x64:).*"            = "`${1} $($Latest.URL64)"
+          "(?i)(checksum64:).*"        = "`${1} $($Latest.Checksum64)"
+        }
   }
 }
 
-function global:au_BeforeUpdate {
-  Get-RemoteFiles -Purge -NoSuffix
-
-  Set-Alias 7z $Env:chocolateyInstall\tools\7z.exe
-  7z e tools\*.zip -otools *.exe -r -y
-  rm tools\*.zip -ea 0
-}
+function global:au_BeforeUpdate {Get-RemoteFiles -Purge}
```

Comment:
Use the form and style of the previous code - don't put a function, and it's code on the same line.